src/Plugin/Filter/FilterVisualContent.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * @file
+ * Filter that enables to use patterns for create visual content more easy.
+ * Drupal\visual_content_layout\Plugin\Filter\FilterVisualContent.
+ *
+ */
+
+namespace Drupal\visual_content_layout\Plugin\Filter;
+
+use Drupal\filter\FilterProcessResult;
+use Drupal\filter\Plugin\FilterBase;
+
+
+/**
+ * Provides a filter to use shortdoces.
+ *.
+ *
+ * @Filter(
+ *   id = "filter_visualcontent",
+ *   title = @Translation("Filtrate Visual Content"),
+ *   description = @Translation("Use shortcodes to generate a easy content."),
```

Comment:
Change this line to read 
*   description = @Translation("Provides a ShortCode filter format to easily generate content layout."),